src/StackExchange.Redis/PhysicalBridge.cs

```diff
@@ -909,7 +913,7 @@ private async Task ProcessBacklogAsync()
                     // TODO: vNext handoff this backlog to another primary ("can handle everything") connection
                     // and remove any per-server commands. This means we need to track a bit of whether something
                     // was server-endpoint-specific in PrepareToPushMessageToBridge (was the server ref null or not)
-                    await ProcessBridgeBacklogAsync(); // Needs handoff
+                    await ProcessBridgeBacklogAsync().ConfigureAwait(false); // Needs handoff
```

Comment:
Use `.ForAwait()` instead of `.ConfigureAwait(false)`?

Reply:
The new backlog code added quite a few other awaited calls without `.ForAwait()`. Do those need to be updated too?

Reply:
This isn't actually relevant either way - I was testing an allocation thing - can be removed entirely (we kick it off without a context in a thread with none). Can tidy as a follow-up though!